### PR TITLE
Upgrade to Node 20

### PR DESCRIPTION
**Pull request by pamella**
## Description

Upgrade to Node 20

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires documentation updates.
- [ ] I have updated the documentation accordingly.
- [X] My change requires dependencies updates.
- [X] I have updated the dependencies accordingly.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ jobs:
     name: Generate stable boilerplate build
     strategy:
       matrix:
-        python-version: [3.8]
-        node-version: [14.5]
+        python-version: [3.12]
```
Comment (pamella) [inline .github/workflows/deploy.yml:14]: Drive-by change to match `proj_main.yml`.

**Review by fjsj — APPROVED** [review]
